### PR TITLE
[v2] small refactor of api-runner-browser to fix loader test

### PR DESCRIPTION
**Pull request by pieh**

### CODE REVIEW

File: packages/gatsby/cache-dir/api-runner-browser.js [hunk 1]
```diff
@@ -1,15 +1,4 @@
...
-//   },
-//   {
-//     plugin: require("/path/to/plugin2/gatsby-browser.js"),
-//     options: { ... },
-//   },
-// ]
+const plugins = require(`./api-runner-browser-plugins`)
 
 exports.apiRunner = (api, args, defaultReturn) => {
   // eslint-disable-next-line no-undef
```
[the remark this answers is not in the document]
Comment (pieh) [reply packages/gatsby/cache-dir/api-runner-browser.js]: You're right, thanks!

**Comment by m-allanson** [comment]
Travis didn't get any output, I've restarted it.

**Comment by pieh** [comment]
@m-allanson Yeah, I did restart it earlier too - I saw same issue, not sure why this is happening

**Comment by m-allanson** [comment]
screen shot 2018-05-09 at 17 55 57

There's an issue with the UI tests, but I think that's separate.

**Comment by KyleAMathews** [comment]
👍

**Pull request merged**